ipynb/energy/EnergyModel_ClusterEnergy.ipynb

```diff
@@ -106,12 +106,12 @@
     "    \"emeter\" : {\n",
     "        'instrument' : 'hwmon',\n",
     "        'conf' : {\n",
-    "            'sites' : [ 'a53', 'a57' ],\n",
+    "            'sites' : [ 'BOARDLITTLE', 'BOARDBIG' ],\n",
```

Comment:
I use Juno R0 board and Linux kernel 3.18 ( Linaro deliverables for Juno 16.06 or 16.09 )

I ever thought "juno" is for Juno R0 board and "juno2" is for Juno R2 board.
If it is not that case, we can use "juno2" in my_conf to bypass this issue. 
But it may cause some confusion for other customers if they have same misunderstanding.